### ISSUE TITLE
allow input of negative sign first before entering a number?

### ISSUE THREAD

**Issue by kkpenas** [cesarrew/ng2-currency-mask#121]
I wonder if you could modify to allow entering negative sign first before the numbers?

**Comment by christopheelkhoury** [comment]
I could take care of this issue @cesarrew

**Comment by kalyanvedurumudi** [comment]
@christopheelkhoury 

Hello, any update on this.

**Comment by christopheelkhoury** [comment]
@kalyanvedurumudi author never gave me authorization to work on this.

**Comment by guillep2k** [comment]
> @kalyanvedurumudi author never gave me authorization to work on this.

The author doesn't seem to have time for the project since June 2018. Being this an open-source project and everything, perhaps someone would like to pick up the glove and create their own fork?

**Comment by christopheelkhoury** [comment]
@guillep2k great idea. I've forked it [right here](https://github.com/christopheelkhoury/ng2-currency-mask). If anyone would like to filter the issues open here and open them on my fork, we could get this wheel rolling.

**Comment by christopheelkhoury** [comment]
@guillep2k You could become one. There's no harm in that. Could help the community and help your expertise at the same time.

**Comment by guillep2k** [comment]
> @guillep2k You could become one. There's no harm in that. Could help the community and help your expertise at the same time.

I'm not even using Angular! :-)
Anyway, I can do that with the open issues, but can't with the pull requests. I'll drop a line on those.

**Comment by guillep2k** [comment]
@christopheelkhoury I'm unable to open issues. There's no such option in the fork. Perhaps you need to change some configuration?

**Comment by guillep2k** [comment]
@christopheelkhoury Yes, you need to go to the "Settings" tab and check the "Issues" box option so I can add the issues.

**Comment by christopheelkhoury** [comment]
@guillep2k done

**Comment by joevandebilt** [comment]
I know that the issue will remain open while the original author isn't working but I was curious if @thelebdev has a workaround for this. I've checked your fork of this project and cannot see any future commits or resolution on this issue - did you include this into your fork and was it ever resolved? 

Thanks

**Comment by thelebdev** [comment]
Hey @joevandebilt buddy, I mentioned [here](https://github.com/cesarrew/ng2-currency-mask/issues/121#issuecomment-494401172) that I've forked the repo and would like to kick-start support for it from a different perspective. I've merged a PR that was opened there recently.

If you want to re-kickstart development on this package, let me know
